mappings/net/minecraft/client/gui/screen/world/WorldListWidget.mapping

```diff
@@ -1,38 +1,57 @@
 	FIELD f_bvfzqfwz CONVERSION_TOOLTIP_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_giqqdqgq levels Ljava/util/List;
 	FIELD f_gmbxrzpw FROM_NEWER_VERSION_FIRST_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_gmmkqxli SNAPSHOT_SECOND_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_iwchirdh loadingEntry Lnet/minecraft/unmapped/C_wwgrvynj$C_chlqbstq;
+	FIELD f_lejfmpml query Ljava/lang/String;
 	FIELD f_mbtquola parent Lnet/minecraft/unmapped/C_yrypplpi;
 	FIELD f_neodhkus FROM_NEWER_VERSION_SECOND_LINE Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_nhmjctua LOGGER Lorg/slf4j/Logger;
 	FIELD f_rtudrfdo LOCKED_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
 	FIELD f_rxeuofpy WORLD_SELECTION_LOCATION Lnet/minecraft/unmapped/C_ncpywfca;
 	FIELD f_xjlsplba UNKNOWN_SERVER_LOCATION Lnet/minecraft/unmapped/C_ncpywfca;
-	METHOD m_adqicvjj narrateScreenIfNarrationEnabled ()V
-	METHOD m_bhyykuuh shouldShow (Ljava/lang/String;Lnet/minecraft/unmapped/C_wfuumzdp;)Z
-		ARG 1 search
+	METHOD <init> (Lnet/minecraft/unmapped/C_yrypplpi;Lnet/minecraft/unmapped/C_ayfeobid;IIIIILjava/lang/String;Lnet/minecraft/unmapped/C_wwgrvynj;)V
+		ARG 1 parent
+		ARG 2 client
+		ARG 3 screenWidth
+		ARG 4 screenHeight
+		ARG 5 top
+		ARG 6 bottom
+		ARG 7 itemHeight
+		ARG 8 query
+		ARG 9 previousList
+	METHOD m_adqicvjj narrateScreen ()V
+	METHOD m_bhyykuuh worldNameMatches (Ljava/lang/String;Lnet/minecraft/unmapped/C_wfuumzdp;)Z
+		ARG 1 query
 		ARG 2 summary
+	METHOD m_egkdspch finalizeLevels ()Ljava/util/List;
 	METHOD m_fapsoaqi getSelectedAsOptional ()Ljava/util/Optional;
-	METHOD m_gojurhkm loadLevels ()Ljava/util/concurrent/CompletableFuture;
-	METHOD m_irbmeiln showLoadingScreen ()V
-	METHOD m_jtfjknzc showSummaries (Ljava/lang/String;Ljava/util/List;)V
-		ARG 1 search
-		ARG 2 summaries
+	METHOD m_gojurhkm generateListFuture ()Ljava/util/concurrent/CompletableFuture;
+	METHOD m_hexfpxkc filterLevels (Ljava/util/List;)V
+		ARG 1 levels
+	METHOD m_irbmeiln resetEntries ()V
+	METHOD m_jtfjknzc filter (Ljava/lang/String;Ljava/util/List;)V
+		ARG 1 query
+		ARG 2 levels
 	METHOD m_lmtvmxlw getParent ()Lnet/minecraft/unmapped/C_yrypplpi;
-	METHOD m_qcspbwca showUnableToLoadScreen (Lnet/minecraft/unmapped/C_rdaqiwdt;)V
-		ARG 1 message
-	CLASS C_chlqbstq LoadingEntry
+	METHOD m_pfgywwta filter (Ljava/lang/String;)V
+		ARG 1 query
+	METHOD m_qcspbwca failedLoadingWorld (Lnet/minecraft/unmapped/C_rdaqiwdt;)V
+		ARG 1 reason
+	METHOD m_qscakuku regenerateList ()V
+	CLASS C_chlqbstq LoadingWorldEntry
 		FIELD f_vrjbextc client Lnet/minecraft/unmapped/C_ayfeobid;
-		FIELD f_zcqqkjsr LOADING_LIST_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
+		FIELD f_zcqqkjsr LOADING_WORLD_LIST_TEXT Lnet/minecraft/unmapped/C_rdaqiwdt;
 		METHOD <init> (Lnet/minecraft/unmapped/C_ayfeobid;)V
 			ARG 1 client
-	CLASS C_hdxtychi Entry
+	CLASS C_hdxtychi AbstractWorldEntry
 		METHOD close close ()V
-		METHOD m_ogqmiqlf isSelectable ()Z
-	CLASS C_jmifisfh WorldEntry
+		METHOD m_ogqmiqlf selectable ()Z
```

Comment:
```suggestion
		METHOD m_ogqmiqlf isSelectable ()Z
```

Reply:
i prefer without the `is` id only use that in getters

Reply:
Ok, so long as the method is not a getter